src/components/v2/Table/index.tsx

```diff
@@ -97,17 +100,17 @@ export const Table = ({
     }
 
     /* if gridTemplateColumns prop is not passed from parent component, we create similar fractions by default */
-    return isSmDown
+    return isCardLayout
       ? /* getting default gridTemplateColumns string depending on columns array length */
         getTemplateColumnsString(columns.slice(1, columns.length))
       : getTemplateColumnsString(columns);
-  }, [columns, gridTemplateColumns]);
+  }, [columns, gridTemplateColumns, isCardLayout]);
 
   return (
-    <Paper css={[styles.root, isSmDown && styles.rootMobile]} className={className}>
-      {title && <h4 css={[styles.title, isSmDown && styles.titleMobile]}>{title}</h4>}
+    <Paper css={[styles.root, isCardLayout && styles.rootMobile]} className={className}>
+      {title && <h4 css={[styles.title, isCardLayout && styles.titleMobile]}>{title}</h4>}
 
-      {isSmDown ? (
+      {isCardLayout ? (
```

Comment:
As we spoke about earlier this week, we want to move away from using Javascript and use CSS instead. You can render both variants and use a CSS media query to hide one or the other using `display: none`.

Reply:
i think, hook use is justified here.
we have different layouts for this cases. that's such hooks are made for.